### PR TITLE
GL-19 api endpoint to delete a warehouse

### PR DESCRIPTION
**Pull request by Sophia-G-Chan**
I used the controller file to have the delete function there and kept the router file more simple. Open to changing the controller file if we want to only have a router file.

### CODE REVIEW

File: routes/warehouse.js [hunk 1]
```diff
@@ -23,5 +23,7 @@ router.use('/:id/inventories', async (req, res)=>{
     }
 });
 
+router
```
Comment (Muudy12) [inline routes/warehouse.js:29]: Not sure if i'm seeing this right but here should start with: router.route(path).delete(etc)

**Review by Muudy12 — CHANGES_REQUESTED** [review]
I added a comment to your route js file. Hope you can see it.

File: controllers/warehouse-controllers.js [hunk 2]
```diff
@@ -17,7 +17,7 @@ const deleteWarehouse = async (req, res) => {
             .where({ id: warehouseID })
             .del()
 
-        res.status(200).json({message: 'Warehouse deleted'})
+        res.status(204).send
```
Comment (Muudy12) [inline controllers/warehouse-controllers.js:33]: Here I am seeing 'rest.status(204).send' only. Did you mean to end it with just send?

**Review by Muudy12 — COMMENTED** [review]
Let a comment again sorry hehe

File: controllers/warehouse-controllers.js (hunk 2 above)
Comment (Sophia-G-Chan) [reply controllers/warehouse-controllers.js:33]: Yeah I read the ticket again and it said to only send 204 with no message

**Review by Muudy12 — APPROVED** [review]
Looks good to me.

**Pull request merged**